### DISCUSSION TITLE
repo problems

### DISCUSSION THREAD

**Discussion by brok-en0810** [termux/termux-packages#11637] [Q&A] [answered]
hi I'm rather new to all this.. but everytime I try to clone a repo I get an error.. it asks fo4 my user name.. I put it in.  it asks for my password witch I know is the auth code. I put that in and I get the error repo can't befound. now it's on every repo I try.. not just 1

**Comment by sylirre** [comment, marked as answer]
Check the repo URL for typos. Make sure it is not truncated, which is pretty common mistake on Termux.

GitHub does not distinguish private repository from nonexistent (incorrectly specified) and asks you for providing a password. Note that if repository is a private and is not yours, you will be asked for owner user name and password, not yours.
